x-pack/plugins/reporting/public/management/report_listing.tsx

```diff
@@ -512,19 +550,25 @@ export const ReportListing = (
     'ilmPolicyContextValue' | 'intl' | 'apiClient' | 'capabilities' | 'configAllowsImages'
   >
 ) => {
-  const ilmPolicyStatusValue = useIlmPolicyStatus();
   const { apiClient } = useInternalApiClient();
   const {
     services: {
       application: { capabilities },
     },
   } = useKibana();
-  return (
+  return props.config.statefulSettings.enabled ? (
+    <ReportListingUi
```

Comment:
This will need to be broken up, so that it switches between calling 2 different components in the conditional, rather than calling the same component 2 different ways in the conditional :)

Reply:
Thank you for the help over zoom. Note for my future self with having two different components called in the ternary: 
```
  return props.config.statefulSettings.enabled ? (
    <ReportListingStateful {...props} apiClient={apiClient} capabilities={capabilities} />
  ) : (
    <ReportListingDefault {...props} apiClient={apiClient} capabilities={capabilities} />
  );
```